src/openms/source/ANALYSIS/MAPMATCHING/QTClusterFinder.cpp

```diff
@@ -99,6 +100,132 @@ namespace OpenMS
     // update parameters (dummy)
     setParameters_(1, 1);
 
+    if(use_IDs_)
+    {
+      std::unordered_map<String, std::vector<double>> ided_feat_rts;
```

Comment:
add comment: map string "modified sequence/charge" to all RTs the feature has been observed in the different maps